Classes/Twig/Typo3Loader.php

```diff
@@ -82,7 +88,14 @@ private function findTemplate(string $name, bool $throw = false): string
             throw new \Twig_Error_Loader($this->errorCache[$name]);
         }
 
-        $path = GeneralUtility::getFileAbsFileName($name);
+        $templatePath = array_values($this->cache)[0];
```

Comment:
The first value in `$this->cache` will contains a random element. To be able to load templates by their relative file name we should add the `\Twig_Loader_Filesystem` to the loader chain I think.